### DISCUSSION TITLE
Allow player to go over budget?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#612] [Ideas]
I wonder if it might be a good idea to allow a player to slightly* exceed their budget under some circumstances - if, for instance, your prestige is lower than your opponent's.  I'm in the middle of one of those "just can't catch a break" games, flying direct ascent for the first time in a long while, and I've had to delay my first launch by a whole year because I'm **1MB** short.  So frustrating!

![1MB short](https://user-images.githubusercontent.com/3118014/149828125-da56578c-2772-4396-a9eb-e4260923e03f.png)

| * I mean by like 1 or 2 MB.  Or perhaps it could depend on your most advanced spacecraft: 1 for Gemini, 2 for Apollo, 3 for XMS-2, 4 for Jupiter.

**Comment by rnyoakum** [comment]
I've heard there are [things you can do](https://github.com/raceintospace/raceintospace/commit/5413507ee3ff7d027f5b691ae8cf050da2522fd7) in Purchasing... 😉

**Reply by peyre** [reply]
Well yes, but without cheating.  in this scenario any deficits would count against your next year's budget.  Potentially with interest.
